Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/AnomalyDescriptor.java

```diff
@@ -40,9 +40,14 @@
     // information
     DiVector attribution;
 
+    long numberOfUpdates;
+
     // timestamp (basically a sequence index); kept as long for potential future use
```

Comment:
Given that we use sequence index in the forest API, we need to explain somewhere (I don't think it necessiarly should be in an inline comment here) why we have both and what the difference is.

Reply:
Its the timestamp example where we can use the raw time field as part of RCF analysis